src/packages.ts

```diff
@@ -94,11 +87,35 @@ export class PackageManager {
 
         pkg.tmpFile = tmpResult;
 
-        const result = await downloadFile(url, pkg, logger, status, proxy);
+        const result = await downloadFile(latestSnootyReleaseUrl, pkg, logger, status, proxy);
         logger.appendLine(' Done!');
 
         return result;
     }
+
+    public async getLatestParserUrl(platform: string, logger: Logger) {
+        const githubUrl = `https://github.com/mongodb/snooty-parser/releases/latest`;
+        const response = await fetch(githubUrl, { headers: { Accept: 'application/json' } });
+        const json = await response.json() as { [k:string]: string };
+    
+        let latestTag = json?.tag_name;
+        if (!latestTag) {
+            logger.appendLine('Error accessing newest snooty-parser version.');
+            latestTag = 'v0.14.10';
```

Comment:
It would be nice if without internet or if we get a weird response, if we could get the most recent version of the parser we've downloaded.

Not part of this PR. This is okay for now!

Reply:
I put in a half-measure of sorts!